deploy/cd-namespace/chart/values.yaml

```diff
@@ -6,41 +6,66 @@
 serviceAccountName: 'pipeline'
 
 # Bitbucket
+# Bitbucket URL (including scheme). Example: https://bitbucket.example.com.
 bitbucketUrl: ''
+# Bitbucket username. Example: cd_user.
 bitbucketUsername: ''
+# Bitbucket personal access token (PAT) in clear text. This token needs to be
+# created in Bitbucket for the user identified by "bitbucketUsername".
```

Comment:
What project and repo permissions should have the token? e.g. `REPO_ADMIN`?

Reply:
275937e